css/at-rules/media.json

```
@@ -1344,8 +1344,7 @@
                   "version_removed": "79"
                 },
                 "firefox_android": {
-                  "version_added": "67",
-                  "version_removed": "79"
+                  "version_added": false
```

I was wrong about this. I'll add a suggestion here to revert this and apply it, since it was my fault to begin with. 😄

```suggestion
                  "version_added": "67",
                  "version_removed": "79"
```